desc/equilibrium.py

```diff
@@ -807,6 +818,7 @@ def _print_iteration(self, ii, equil):
         equil.resolution_summary()
         print("Boundary ratio = {}".format(self.inputs[ii]["bdry_ratio"]))
         print("Pressure ratio = {}".format(self.inputs[ii]["pres_ratio"]))
+        print("Current ratio = {}".format(self.inputs[ii]["curr_ratio"]))
```

Comment:
maybe only print this if its in fixed current mode? otherwise might be confusing